vm/devices/tpm/src/lib.rs

```diff
@@ -1456,10 +1456,27 @@ mod save_restore {
         TpmRuntimeLib(#[source] ms_tpm_20_ref::Error),
     }
 
+    #[derive(Error, Debug)]
+    pub enum TpmSaveError {
+        #[error("servicing is blocked when there is an outstanding AK Cert request")]
+        OutstandingAkCertRequest,
+    }
+
     impl SaveRestore for Tpm {
         type SavedState = state::SavedState;
 
         fn save(&mut self) -> Result<Self::SavedState, SaveError> {
+            // Block servicing requests when there is an outstanding ak cert request.
```

Comment:
Reword this as `block save`, and i'd devnote it as the following:
1. the device itself does not save/restore the async request, and we need to think more about what it means to save this request outstanding and what the API should be.
2. The existing implementation with the GET has a host issue where leaving this request in-flight during a servicing operation can lead to bad host behavior on older hosts.